src/status_im/contexts/profile/edit/name/events.cljs

```diff
@@ -0,0 +1,16 @@
+(ns status-im.contexts.profile.edit.name.events
+  (:require [utils.i18n :as i18n]
+            [utils.re-frame :as re-frame]))
+
+(defn edit-profile-name
+  [{:keys [db]} [name]]
+  {:db            (assoc-in db [:profile/profile :display-name] name)
+   :json-rpc/call [{:method     "wakuext_setDisplayName"
```

Comment:
Forgot to comment @mohsen-ghafouri, but for new code and for consistency sake, it would be best if we could stick to our guideline https://github.com/status-im/status-mobile/blob/d495bb6b572f86511caa67477d5022472ab46e1d/doc/new-guidelines.md#registering-event-handlers (and from re-frame) to use `:db` and `:fx` only as the keys of the effects map.